lib/puppet/provider/cisco_aci_fvaepg/cisco_aci.rb

```diff
@@ -1,5 +1,5 @@
 #
-# May 2018
```

Comment:
We should not include date changes in the diffs if there is not change to the code.

Reply:
Reverted the date changes.